RecoMET/METFilters/python/BadChargedCandidateFilter_cfi.py

```diff
@@ -10,6 +10,7 @@
     minPtDiffRel = cms.double(0.00001),               # lower threshold on difference between pt of reco::muon->innerTrack and pfCandidate
```

Comment:
Hi @slava77 
I am getting following error if I change every cfi (*) assosciated with BadParticleFilter.cc  in the above format.
Shouldn't _mod be defined as was done  here:  https://github.com/amkalsi/cmssw/blob/11_1_X-MET0filter/RecoMET/METFilters/python/BadPFMuonDzFilter_cfi.py

Please let me know

(*)
  BadPFMuonDzFilter = _mod.badParticleFilter.clone(
NameError: name '_mod' is not defined
Failure using cmsdriver 94X v1: status 1
status = 256

Reply:
Yes, the config from which you are importing must be defined.
It will be
  RecoMET.METFilters.BadParticleFilter_cfi.py
as soon as you apply the suggestion of Slava in https://github.com/cms-sw/cmssw/pull/30015/files#r451684411

Reply:
sorry, I missed the import line when repeating the earlier comment as shown in https://github.com/cms-sw/cmssw/pull/30015#discussion_r437772069
```python
import  RecoMET.METFilters.BadPFMuonFilter_cfi as _mod
```